data/common.yaml

```diff
@@ -1,6 +1,8 @@
 ---
 kibana::ensure: present
 kibana::config: {}
+kibana::kibana_user: kibana
+kibana::kibana_group: kibana
```

Comment:
Please remove this (see previous comment)

Reply:
ok